fixtures/build/dir012/composer.json

```diff
@@ -15,7 +15,10 @@
         "preferred-install": {
             "*": "dist"
         },
-        "sort-packages": true
+        "sort-packages": true,
+        "platform": {
+            "php": "7.2.9"
```

Comment:
Should we bump it to 7.3?

Reply:
Let us have a last release for PHP 7.2 with an uploaded phar-file ;-)